schemata_json_schema/src/Normalizer/jsonapi/DataDefinitionNormalizer.php

```diff
@@ -79,15 +72,48 @@ public function normalize($entity, $format = NULL, array $context = array()) {
    *   Serializer context.
    *
    * @return array
-   *   Discrete values of the property definition.
-   *
-   * @todo identify how to cleanly inject the plugin manager without requiring
-   *   updates to many of the normalizers.
+   *   Discrete values of the property definition
    */
   protected function extractPropertyData(DataDefinitionInterface $property, array $context = []) {
-    $type_mapper_manager = \Drupal::service('plugin.manager.json_schema.type_mapper');
-    $data = $type_mapper_manager->createInstance($property->getDataType())
-      ->getMappedValue($property);
```

Comment:
Looks like this plugin mechanism was dropped from the json described format's version of this method. It's needed for optional types, such as Metatag.